config/locales/views/user_sessions/new/en.yml

```diff
@@ -3,7 +3,7 @@
 en:
   user_sessions:
     new:
-      log_in_or_sign_up: "Log in <small> or <a href='%{url1}'>sign up to join</a> the Public Lab community</small>"
+      log_in_or_sign_up: "Log in <small> or <a>sign up to join</a> the Public Lab community</small>"
```

Comment:
Can you please explain these changes?

Reply:
I removed the URL so that when I click "sign up to join" in the modal, it will change it's contents to the signup modal instead of also loading up the /signup page, which would close the modal.